modules/tinymce/src/core/test/ts/browser/selection/GetSelectionContentTest.ts

```diff
@@ -22,20 +22,23 @@ UnitTest.asynctest('browser.tinymce.selection.GetSelectionContentTest', (success
     input.parentNode.removeChild(input);
   });
 
-  const sAddTestDiv = Step.sync(function () {
+  const sAddTestDiv = Step.sync(() => {
     const div = document.createElement('div');
     div.innerHTML = 'xxx';
     div.contentEditable = 'true';
     div.id = testDivId;
     document.body.appendChild(div);
   });
 
-  const cGetContent = (args: any) => Chain.mapper((editor: Editor) => GetSelectionContent.getContent(editor, args));
+  const cGetContent = (args: any) =>
```

Comment:
This could be typed now, as we have the `GetSelectionContentArgs` type. Only mentioning this, as you've changed this line.